src/AccordionPanel.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import cx from 'classnames';
+
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+import Icon from './Icon';
+
+/**
+ * @module AccordionPanel
+ */
+class AccordionPanel extends React.Component {
+	/**
+	 * Document this for PR
+	 */
+	constructor(props){
+		super(props);
+		this.state = {
+			open: this.props.isOpen
+		};
+
+		this._handleToggle = this._handleToggle.bind(this);
+		this._updateHeight = this._updateHeight.bind(this);
+	}
+
+	/**
+	 * Document this for PR
+	 */
+	_handleToggle(event){
+		this.setState({
+			open: !this.state.open
+		});
+	}
+
+	_updateHeight() {
+		const content = ReactDOM.findDOMNode(this.content);
```

Comment:
Using ReactDOM.findDOMNode() is considered a bad in the context of a component is considered a bad practice. Since `this.content` is already a reference to the DOM element you should be able to access the elements height. You want to go with https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect instead.